datafusion/proto/tests/cases/roundtrip_logical_plan.rs

```diff
@@ -581,7 +581,7 @@ async fn roundtrip_expr_api() -> Result<()> {
             make_array(vec![lit(1), lit(2), lit(3)]),
             lit(1),
             lit(2),
-            lit(1),
+            Some(lit(1)),
```

Comment:
Maybe add the example that has None for stride?

Reply:
Added.